versions.tf

```
@@ -7,6 +7,9 @@ terraform {
   required_version = ">= 0.14, < 2.0"
 
   required_providers {
-    google-beta = "~> 3.60"
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 4.0, < 5.0"
```

```suggestion
      version = ">= 3.60, < 5.0"
```

can't we keep the support for provider 3.6?

---

yes, looks like there is no code change? @thiesen what is the reason?

---

the reason was me being silly - I think this was the first `google-beta` update that I got and I made a confusion :man_facepalming: